client/src/common_text/common_lang.yaml

```diff
@@ -279,6 +282,9 @@ open_data_datasets:
 link:
   en: Link
   fr: Lien
+the_goc:
+  en: the Government of Canada
+  fr: le gouvernement du Canada
```

Comment:
I think the G was meant to be capital in the french version too

Reply:
Double checked, other uses in the IB have lower case g in French